xsd/siri_model/siri_situationAffects-v2.0.xsd

```diff
@@ -646,37 +641,11 @@ Rail transport, Roads and road transport
    <xsd:documentation>Type for information about the SECTIONs affected by a SITUATION.</xsd:documentation>
   </xsd:annotation>
   <xsd:sequence>
-   <xsd:choice>
    <xsd:element name="SectionRef" type="SectionRefStructure" minOccurs="0">
     <xsd:annotation>
      <xsd:documentation>Reference to a section of ROUTE affected by a SITUATION.</xsd:documentation>
     </xsd:annotation>
    </xsd:element>
-   <xsd:element name="IndirectSectionRef">
```

Comment:
Is this removal of the choice IndirectSectionRef intentional?
(just asking - I don't remember the latest discussion on this)

Reply:
Yes, the implementation with the UmS Group is still being discussed. Therefore I've separated the pullrequests last week (see https://github.com/SIRI-CEN/SIRI/pull/68) so that we can merge this PR without the AffectedRoute/AffectedSection additions

Reply:
Fine for me. :-)